generate.py

```diff
@@ -522,7 +532,7 @@ def tagify_object(object_id, taxonomies):
     delete_tags_from_taxonomy(
         lightcast_skills_taxonomy,
         list(map(lambda t: t["value"], lightcast_skills_taxonomy_tags)),
-        lightcast_skills_taxonomy_tags
+        with_subtags=True,
```

Comment:
I'm now sure why these calls were different from the others (and this was raising and lint error because `with_subtags` is boolean.

Not sure if the solution here is this or `with_subtags=bool(lightcast_skills_taxonomy_tags)`.

Reply:
`with_subtags=True` is fine.